server/routes/index.js

```diff
@@ -11,8 +11,8 @@ router.get('/', function (req, res) {
 
 router.post('/', function (req, res) {
   const post = req.body.url
-  db.addPost(post, (err, postUrl) => {
-    res.json({message: 'post added successfully'})
+  db.addPost(post, (err, post) => {
+  res.status(201).json(post)
```

Comment:
Normally a `201` doesn't have a response body, so you can remove the `.json(post)` part.